File: LANGUAGES/1) C++/2) INTEGER/11.cpp
#include <iostream>

using namespace std;

int main()
{
	int n, yuz, on, bir, S;
	cout << " ABC = ";
	cin >> n;
	yuz = n / 100;
	on = n / 10 % 10;
	bir = n % 10;
	S = yuz + on + bir;
	cout << " Raqamlari yigindisi = " << S;
	return 0;
}

File: LANGUAGES/4) Php/2) INTEGER/11.php
<?php
$n = 578;
echo " N = " . $n . " <br>";
$yuz = intval($n / 100);
$on = intval($n / 10) % 10;
$bir = $n % 10;
$yig = $yuz + $on + $bir;
echo " Raqamlar yigindisi = " . $yig;